spec/services/parsers/jpg_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe Parsers::Jpg do
+  subject { described_class.new(document) }
+
+  let(:chunking_profile) { create(:chunking_profile, method: :basic, size: 800) }
+  let(:file) { fixture_file_upload("avatar.jpg", 'application/image') }
+  let(:document) { create(:document, state: :created, file:, chunking_profile:) }
+
+  context "when method is :basic" do
+    it_behaves_like "all parsers"
+
+    it "succeeds basic chunking" do
+      expect(subject.chunks.count).to eq(17)
```

Comment:
I'm having trouble getting this to succeed.  The site seems to work fine and creates right number of chunks for this file based on size, but fails here.  I added a 'chunk count' text to other classes and they succeed, but not here.  Kinda like `fixture_file_upload` isn't working for jpg.